src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -247,6 +247,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 metadataImportOptions:=other.MetadataImportOptions,
                 referencesSupersedeLowerVersions:=other.ReferencesSupersedeLowerVersions,
                 publicSign:=other.PublicSign)
+            _ignoreCorLibraryDuplicatedTypes = other.IgnoreCorLibraryDuplicatedTypes
```

Comment:
Consider adding a `ignoreCorLibraryDuplicatedTypes` parameter to other constructor instead, for consistency with other fields, unless that leads to a significant number of changes. (Are there any references to the other constructor outside of this class? Could that constructor be `Private`?)